demo/rt-detr-object-detection/requirements.txt

```
@@ -0,0 +1,4 @@
+safetensors==0.4.3
+opencv-python
+torch
+transformers
```

since `rt-detr` was recently added
```suggestion
transformers>=4.43.0
```